styles/lichess/catppuccin.user.css

```diff
@@ -0,0 +1,567 @@
+/* ==UserStyle==
+@name lichess Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/lichess
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/lichess
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/lichess/catppuccin.user.css
+@description Soothing pastel theme for lichess
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select flavor "Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater*", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+@-moz-document domain("lichess.org") {
+  :root {
+    #catppuccin(@flavor, @accentColor);
+  }
```

Comment:
> Looks like there is a light/dark/auto theme toggle, body[data-theme="x"]. Possible values are "light", "dark", and "system" - you could do something like:

```suggestion
@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
@var select accentColor "Accent" ["rosewater:Rosewater*", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
==/UserStyle== */
@-moz-document domain("lichess.org") {
  body[data-theme="light"] {
    :root {
      #catppuccin(@lightFlavor, @accentColor);
    }
  }
  body[data-theme="dark"] {
    :root {
      #catppuccin(@darkFlavor, @accentColor);
    }
  }
  body[data-theme="system"] {
    @media (prefers-color-scheme: light) {
      :root {
        #catppuccin(@lightFlavor, @accentColor);
      }
    }
    @media (prefers-color-scheme: dark) {
      :root {
        #catppuccin(@darkFlavor, @accentColor);
      }
    }
  }
```
I tried doing this, it doesn't seem like it works tho? Not sure what i'm doing wrongg

Reply:
Oh yeah lol since the `:root` would be inside the `body[data-theme="dark"]`...

Reply:
Aha! You can do `:root:has(body[data-theme="x"])`!

Reply:
woo!